polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -367,10 +366,6 @@ $loading-panel-height: 53px;
 
 .TableHeading-sortable {
   background: var(--p-color-bg-subdued);
-
-  #{$se23} & {
-    background: var(--p-color-bg-secondary-experimental);
```

Comment:
Do we still need this to ensure the specificity of later selectors don't try to set it back to the white colour again? I only vaguely remember working on this and we fixed some edge cases with rules like this.